src/ast.js

```diff
@@ -3,24 +3,50 @@ const {symbolMap} = require('./constants');
 
 function traversSourceCode(source, parse) {
     const root = AST.parse(source, parse);
+    const sourceLines = source.split('\n');
```

Comment:
Move this variable into `getSrc` function

Reply:
This will create an unwanted overhead when there are a lot of nodes.
Most nodes are subnodes, and the getSourceByLines function ends up being called a lot.
I think that instead of splitting into lines each time we can split it ocne and re-use it.